fswebcam.c

```diff
@@ -599,7 +599,7 @@ int fswc_grab(fswebcam_config_t *config)
 	config->height = src.height;
 	
 	/* Allocate memory for the average bitmap buffer. */
-	abitmap = calloc(config->width * config->height * 3, sizeof(avgbmp_t));
+	abitmap = calloc(config->width * config->height * 4, sizeof(avgbmp_t));
```

Comment:
This change isn't needed.

Reply:
ok, but why do you allocate memory for the 'average buffer' instead of allocating exactly the amount of memory needed ?

Reply:
It's an RGB buffer of type avgbmp_t, so width * height * 3 is exactly what it needs. The A component of ARGB types isn't stored.